app/controllers/api/v1/basic_auth_api/person_controller.rb

```diff
@@ -7,6 +7,24 @@ class PersonController < BasicAuthApiController
         before_action :set_person, only: %i[change_to_mentor]
         before_action :set_mentor_role, only: %i[change_to_mentor]
 
+        def create
+          auth_user = AuthUser.from_token_payload(new_person_params)
+          # This endpoint was added for UMO, where we need to sync users and we need their real emails stored so we can
+          # invite them to fill in questionnaires. AuthUser.from_token_payload creates a new anonymous user (with no
+          # email), so we need to add it manually
+          auth_user.person.update!(email: new_person_params[Rails.application.config.settings.metadata_field][:email])
+
+          render status: :created, json: auth_user
+        rescue ActionController::ParameterMissing => e
+          # Do we have a standardized way of presenting errors?
```

Comment:
We have the same `render_resource`, `validation_error`, `access_denied`, `not_found`, `created`, `destroyed`, `unprocessable_entity`, and `not_found` methods that we have in sdv base platform. They are defined in ApplicationHelper and included in the ApplicationController and so available in all controllers.

Reply:
ah, that's what i was looking for. I remembered this were defined in application_controller.rb on sdv, but after a quick look on vsv i didn't find them. I'll try to them then.